nuxeo-distribution/nuxeo-distribution-resources/src/main/resources/templates-common/common/config/nuxeo.properties

```diff
@@ -30,3 +30,6 @@ org.nuxeo.ecm.platform.liveedit.config=client
 
 # size limit to use the specific download servlet for big files (use SI or IEC prefix)
 #org.nuxeo.big.file.size.limit=5Mi
+
+# default byte prefix format (SI/JEDEC/IEC)
+#org.nuxeo.defaultBytePrefixFormat
```

Comment:
Yes, my bad for omitting the '=SI'. Would you like me to submit another pull request with this modification?